### PR TITLE
style: homepage

### PR DESCRIPTION
**Pull request by bryanlundberg**
**What does this PR do?**
Improve homepage layout

**Related Issue(s)**
No previous issue

**Changes**
 - Applied a gradient color to the title.
 - Made minor revisions to the text content.
 - Implemented a hover effect for the button.
 - Updated the background color using gradient.
 - Eliminated inline styles and transitioned to Tailwind CSS classes.
 - Created a new folder to store images also added few images inside.
 - Improve height page and scroll-bar logic

**Screenshots or GIFs (if applicable)**
Before:
image

After:
image

**Checklist**
- [x] I have read and followed the [contributing guidelines](../CONTRIBUTING.md).
- [x] My code follows the project's coding style and conventions.
- [ ] I have added unit tests (if applicable) to ensure the code works as expected.
- [ ] I have updated the project documentation (if necessary).

**Additional Comments**
Add any additional comments, context, or information about this PR.

**By submitting this PR, I confirm that:**
- [x] I have reviewed my code and believe it is ready for merging.
- [x] I understand that this PR may be subject to review and changes.
- [x] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

File: src/Pages/HomePage.jsx [hunk 1]
```diff
@@ -1,110 +1,106 @@
...
-            <span className='text-[#6F4FF2] font-normal '>SnapGrid</span> is a
-            versatile web platform that simplifies image discovery and
-            downloads. Users can effortlessly find and acquire high-quality
-            images spanning various categories, from nature to architecture.
+
+          <p
+            ref={ref2}
+            className='w-4/5 max-w-2xl leading-8 text-center text-md text-neutral-400 md:text-xl'
+          >
+            Used by some of the world's largest companies,{' '}
```
Comment (Vijaykv5) [inline src/Pages/HomePage.jsx:65]: Remove this as we don't right now used by companies.

Comment (bryanlundberg) [reply src/Pages/HomePage.jsx]: ok

**Comment by bryanlundberg** [comment]
Might fix: #194

**Comment by Vijaykv5** [comment]
May you could add the animation also to the image something like fade-in @bryanlundberg 

As right now all other elements are animating but the image is loading without any effects on it

**Comment by Vijaykv5** [comment]
@bryanlundberg Please don't change the effects of Headers and buttons. Keep it same!
Only image should hae that fade-in animation

**Comment by bryanlundberg** [comment]
I can't do that, there is a rain of different animations and speed transitions also images take a little bit longer load so everything is not synchronized. 

Ends with a strange effect

**Comment by Vijaykv5** [comment]
Oh then I think we can go with the last effect ...not fade-in but before that animate to left..

**Comment by Vijaykv5** [comment]
Are you working on it @bryanlundberg Because I'm prioritizing this thing now

**Comment by bryanlundberg** [comment]
> Are you working on it @bryanlundberg Because I'm prioritizing this thing now

No, I deleted my branch.
Sorry 😐

**Pull request closed without merging**